udata_gouvfr/views.py

```diff
@@ -76,26 +71,17 @@ def dataconnexions():
     return theme.render('dataconnexions.html', categories=categories)
 
 
-@blueprint.route('/faq/', endpoint='faq_home')
-def faq_home():
-    return theme.render('faq/home.html')
-
-
-@blueprint.route('/faq/citizen/', endpoint='faq_citizen')
-def faq_citizen():
-    return theme.render('faq/citizen.html')
-
-
-@blueprint.route('/faq/producer/', endpoint='faq_producer')
-def faq_producer():
-    return theme.render('faq/producer.html')
+@blueprint.route('/faq/', defaults={'section': 'home'})
+@blueprint.route('/faq/<string:section>/')
+def faq(section):
```

Comment:
I can revert this but I find it more logic to route with:
```python
url_for('gouvfr.faq')
url_for('gouvfr.faq', section='citizen')
```

What do you think about it ?